apps/src/lib/ui/SchoolInfoInterstitial.jsx

```diff
@@ -218,7 +218,7 @@ export default class SchoolInfoInterstitial extends React.Component {
       errors.country = true;
       isValid = false;
     } else {
-      if (this.isBlank(schoolType)) {
+      if (this.isBlank(schoolType) && country === 'United States') {
```

Comment:
There is a check below for the country as well. Could we make line 220 `else if (country === 'UnitedStates')`?